src/utils/sanitizedData.js

```diff
@@ -0,0 +1,19 @@
+function sanitizeData(data) {
```

Comment:
improve the function name, this name doesn't make it clear what kind of data you are sanitizing.